src/diffusers/models/resnet.py

```diff
@@ -35,12 +35,15 @@ def __init__(self, channels, use_conv=False, use_conv_transpose=False, out_chann
         else:
             self.Conv2d_0 = conv
 
-    def forward(self, x):
+    def forward(self, x, size=None):
```

Comment:
I think we should save `size` in the init here under `self.interpolation_size` -> it's not really something that should be forwarded to the classes IMO

Reply:
Unfortunately as mentioned in above comment, that would not produce correct results.